src/GDGUkraine/utils.py

```diff
@@ -114,3 +122,283 @@ def make_vcard(user_reg, url=None):
         vcard.format(
             user=user_reg.user, reg=user_reg, event=user_reg.event,
             url=url))
+
+
+def uri_builder(rparams, *args, **kwargs):
+    '''
```

Comment:
If it is a docstring, you should use a triple double-quotes.
https://www.python.org/dev/peps/pep-0257/